doc/modules/metrics.rst

```diff
@@ -33,6 +33,27 @@ the kernel:
     2. ``S = 1. / (D / np.max(D))``
 
 
+.. currentmodule:: sklearn.metrics
+
+.. _pairwise_distances:
+
+The distances between each row vector of ``X`` and a column vector ``y``
```

Comment:
It's between each row of X and each row of Y.
Uppercase Y, please.

Reply:
fixed.